cmd/release/github.go

```diff
@@ -0,0 +1,168 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v25/github"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+)
+
+type app struct {
+	owner  string
+	repo   string
+	client *github.Client
+}
+
+var (
+	contextTimeout = time.Second * 10
+)
+
+func getClient(accessToken string) *github.Client {
+	ctx, cancel := context.WithTimeout(context.Background(), contextTimeout)
+	defer cancel()
+
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{
+			AccessToken: accessToken,
+		},
+	)
+
+	client := github.NewClient(oauth2.NewClient(ctx, ts))
+
+	return client
+}
+
+func new(accessToken string) *app {
+	return &app{
+		owner:  "MeilleursAgents",
```

Comment:
Maybe stored in a constant or a flag with a default value

Reply:
done 5bb2121